httping.go

```diff
@@ -133,10 +137,11 @@ func main() {
 	if jsonResults == false {
 		fmt.Printf("HTTP %s to %s (%s):\n", httpVerb, url.Host, urlStr)
 	}
-	ping(httpVerb, url, *countPtr, hostHeader, jsonResults)
+
+	ping(httpVerb, url, *countPtr, *timeoutPtr, hostHeader, jsonResults, noProxy)
 }
 
-func ping(httpVerb string, url *url.URL, count int, hostHeader string, jsonResults bool) {
+func ping(httpVerb string, url *url.URL, count int, max_timeout int, hostHeader string, jsonResults bool, noProxy bool) {
```

Comment:
Outside this function we're not using the term "max timeout" but just timeout. I think we should keep just timeout without the "max" for consistency and also because there is no other timeout. Thoughts?

Reply:
Hi. Yes, I agree. I didn't want to edit existing code too much, but in this case, I'll name `timeout` inside of the function to `elapsedTime`

Reply:
It's not as simple in Go. Inside the function, `timeout` is an object of type `Time.duration`, and `max_timeout` is an int. 

I think another option is to generate `timeout` value outside of the function. I'll do so, but I'm not sure if it's significantly better.

Reply:
As described above, addressed in https://github.com/pjperez/httping/pull/15/commits/de300469b56483194e5eaac365a106d5fbfad9a6